h2o-genmodel/src/main/java/hex/genmodel/descriptor/VariableImportances.java

```diff
@@ -0,0 +1,27 @@
+package hex.genmodel.descriptor;
+
+import java.io.Serializable;
+import java.util.Arrays;
+
+/**
+ * Represents model's variables and their relative importances in the model.
+ * The structure is model-independent.
+ */
+public class VariableImportances implements Serializable {
+
+    private final String[] _variables;
+    private final double[] _importances;
+
+    public VariableImportances(String[] variableNames, double[] relativeImportances) {
+        this._variables = variableNames;
+        this._importances = relativeImportances;
+    }
+
+    public String[] getVariableNames() {
+        return Arrays.copyOf(_variables, _importances.length);
```

Comment:
Why the copy?

Reply:
This is a leftover and was **removed** in commit `b65f0e1834`. **Thank you for finding it.** At one stage of this PR, I was playing with handing out immutable objects where there is not way of modifying their state by the user. As these structures are potentially customer-facing, I wanted to see what some "best practices" (pushed by limited number of people) look like in action. Turns out there is not only any benefit, but it sucks big time. If people modify these arrays, it's their own stupidity.

Long story short, it is removed together with the getters, which are completely unnecessary right now.

Reply:
This is a leftover and was removed in commit `b65f0e1834`. Thank you for finding it. 
Long story short, it is removed together with the getters, which are completely unnecessary right now.